README.rst

```diff
@@ -105,36 +105,32 @@ Keyring Support
 Instead of typing in your password every time you upload a distribution, Twine
 allows storing a username and password securely using `keyring`_.
 Keyring is installed with Twine but for some systems (Linux mainly) may
-require
-`additional installation steps <https://pypi.org/project/keyring/#installation-linux>`_.
+require `additional installation steps`_.
 
 Once Twine is installed, use the ``keyring`` program to set a
 username and password to use for each package index (repository) to
 which you may upload.
 
-For example, to set a username and password for PyPI:
+For example, to set a username and password for PyPI::
 
-.. code-block:: console
+   $ keyring set https://upload.pypi.org/legacy/ your-username
+
+or ::
 
-    $ keyring set https://upload.pypi.org/legacy/ your-username
-    # or
-    $ python3 -m keyring set https://upload.pypi.org/legacy/ your-username
+   $ python3 -m keyring set https://upload.pypi.org/legacy/ your-username
 
-And enter the password when prompted.
+and enter the password when prompted.
 
 For a different repository, replace the URL with the relevant repository
 URL. For example, for Test PyPI, use ``https://test.pypi.org/legacy/``.
 
 The next time you run ``twine``, it will prompt you for a username and will grab
 the appropriate password from the keyring.
 
-.. Note:: If you are using Linux in a headless environment (such as on a
-    server) you'll need to do some additional steps to ensure that Keyring can
-    store secrets securely. See `Using Keyring on headless systems`_.
+**Note:** If you are using Linux in a headless environment (such as on a server)
```

Comment:
I'd prefer to keep the admonition of this "Note" when it's rendered on Read The Docs:

https://twine.readthedocs.io/en/latest/#keyring-support
https://sphinx-rtd-theme.readthedocs.io/en/stable/demo/demo.html#admonitions

```suggestion
.. Note:: If you are using Linux in a headless environment (such as on a server)
```

Does that cause problems elsewhere?

Reply:
It doesn't get rendered prettily on either [GitHub](https://github.com/pypa/twine#keyring-support) or [PyPI](https://pypi.org/project/twine/), where AFAIK it's read the most.  Do you still want me to revert to the note command?

Reply:
> where AFAIK it's read the most

How do you know that? What hard data leads you to that conclusion?

Reply:
I think it's a fair point that the "Note" isn't rendered clearly GitHub or PyPI, and since Twine uses the same source for those and ReadTheDocs, I think it's makes sense to use the lowest common denominator. IE, I'm okay with using `**Note:**`.